modules/patterns/pages/github-merge-queues.adoc

```diff
@@ -0,0 +1,64 @@
+= GitHub Merge Queues
+
+This document contains instructions for how to configure configure GitHub link:https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue[merge queues] with {ProductName}.
+
+A merge queue is a GitHub feature that provides the same benefits as the **Require branches to be up to date before merging** branch protection, but does not require a pull request author to update their pull request branch and wait for status checks to finish before trying to merge.
+
+By leveraging merge queues, you can be more sure that competing changes landing on your main branch do not have conflicting changes that fail CI, but without the hassle of requiring you to manually rebase every pull request when another one merges.
+
+NOTE: GitHub merge queues are similar to both link:https://docs.prow.k8s.io/docs/components/core/tide/[tide] pools from the link:https://docs.prow.k8s.io/docs/[prow] system and link:https://docs.gitlab.com/ci/pipelines/merge_trains/[merge trains] from Gitlab.
```

Comment:
Should we address these in this document? Or do they need to be addressed as it relates to Konflux?

Reply:
A separate doc sounds reasonable. Does it provide value to have these links in the current page?

Reply:
I think it might -- only because I've explained how to do this a few times to people and almost every time, people said "merge queues? what's that" and, explaining by analogy tended to clear it up pretty quick.

I dunno, as a reader, if it looks like clutter we can definitely drop it.

Reply:
Let's keep it for now. The only issue is the antora error.